### PR TITLE
Sprig Gallery API

### LINKED ISSUE #1435: "New games" card on main hackclub.com website shows invalid dates (and possibly incorrect games)
Screenshot 2024-02-14 at 2 10 22 PM

### PR DESCRIPTION
**Pull request by GalaxyGamingBoy**

### CODE REVIEW

**Comment by GalaxyGamingBoy** [comment]
cc @JosiasAurel @grymmy (I don't have perms to add yall to reviewers so i'll just cc)

File: .gitpod.yml
Comment (JosiasAurel) [inline .gitpod.yml:1]: please remove this gitpod config file

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
lgtm. please remove the gitpod config file

File: pages/api/games.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 export async function getGames() {
   let games = await fetch(
-    'https://raw.githubusercontent.com/hackclub/sprig/main/games/metadata.json'
+    'https://sprig.hackclub.com/api/gallery?new'
   ).then(res => res.json())
-  games = games
-    .sort((a, b) => new Date(b.addedOn) - new Date(a.addedOn))
-    .slice(-4)
 
+  games.length = 4
```
Comment (JosiasAurel) [inline pages/api/games.js]: let's make it clear this is supposed to return just the first 4 games. use a .slice

**Review by JosiasAurel — CHANGES_REQUESTED** [review]

File: pages/api/games.js (hunk 1 above)
Comment (JosiasAurel) [reply pages/api/games.js]: or add a comment explaining what this does

**Review by JosiasAurel — APPROVED** [review]

**Review by grymmy — APPROVED** [review]

**Pull request merged**